runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -371,7 +371,7 @@
     private volatile boolean isShutdown = false;
 
     @Getter
-    private static MetricRegistry defaultMetrics = new MetricRegistry();
+    private static final MetricRegistry defaultMetrics = new MetricRegistry();
     @Getter
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1134)